samples/sampler/stories/default/Picker.js

```diff
@@ -1,34 +1,38 @@
 import {mergeComponentMetadata} from '@enact/storybook-utils';
 import {action} from '@enact/storybook-utils/addons/actions';
 import {boolean, select, text} from '@enact/storybook-utils/addons/knobs';
-import {storiesOf} from '@storybook/react';
 
 import {PickerBase} from '@enact/agate/internal/Picker';
 import Picker from '@enact/agate/Picker';
 
+Picker.displayName = 'Picker';
 const Config = mergeComponentMetadata('Picker', Picker, PickerBase);
 
-storiesOf('Agate', module)
-	.add(
-		'Picker',
-		() => (
-			<div style={{padding: '0 20%'}}>
-				<Picker
-					aria-label={text('aria-label', Config, '')}
-					decrementAriaLabel={text('decrementAriaLabel', Config, '')}
-					disabled={boolean('disabled', Config)}
-					incrementAriaLabel={text('incrementAriaLabel', Config, '')}
-					noAnimation={boolean('noAnimation', Config)}
-					onChange={action('onChange')}
-					orientation={select('orientation', ['vertical', 'horizontal'], Config)}
-					spotlightDisabled={boolean('spotlightDisabled', Config)}
-					wrap={boolean('wrap', Config)}
-				>
-					{['LO', '16\xB0', '17\xB0', '18\xB0', '19\xB0', 'HI']}
-				</Picker>
-			</div>
-		),
-		{
-			text: 'Basic usage of Picker'
-		}
-	);
+export default {
+	title: 'Agate/Picker',
+	component: 'Picker'
+};
+
+export const _Picker = () => (
+	<Picker
```

Comment:
```suggestion
<div style={{padding: '0 20%'}}>
	<Picker
```

Reply:
This would push `Picker` too much to the right. I don't this this is needed anymore.
![picker](https://user-images.githubusercontent.com/63335823/112347434-86a4d380-8ccf-11eb-99be-686f251d610a.PNG)

Reply:
Let's remove some style code in ThemeEnvironment.module.less.
(Line 36 ~ 46)
```
// Here, we target the storybook rendering canvas div. It's inserted as a child of our Panel, but it
// applies position:relative, which doesn't help us when we have stories that should fill the entire
// panel body. This targets that inaccessible node using Panel's structure and storybook's structure
// to update that goofy rule to a rational one.
// The :not() bit targets only non-enact elements, so if a story doesn't use withInfo, and a `div`
// isn't added, the components aren't being told to be a different size/position.
.storyBody > div:not([class]):first-child {
	width: 100%;
	height: 100%;
	position: relative !important;
}
```
We don't need this style code. In storybook 5.x, the storybook inserted an empty div as a child of our Panel for each story. So we needed the above code to make the story shown in the entire Panel body.
But, the storybook doesn't insert it after version 6.
This style code was also removed in the sandstone storybook for the same reason.

After removing the above code, the Picker that styled with padding will be shown properly.
Let me know if it doesn't make the expected result. :)

Reply:
Now it looks ok. Thanks!